users/utils/birthday_resolver.py

```diff
@@ -0,0 +1,136 @@
+import datetime
+
+import requests
+from django.conf import settings
+from helusers.user_utils import get_or_create_user
+from requests import RequestException
+from rest_framework import HTTP_HEADER_ENCODING
+from sentry_sdk import capture_exception
+
+
+def resolve_user(request, payload):
+    user = get_or_create_user(payload, oidc=True)
+
+    # If auth method is in configured loa levels we can try to read the date of birth of the user.
+    if (
+        payload.get("loa", "").lower()
+        in settings.OPEN_CITY_PROFILE_LEVELS_OF_ASSURANCES
+        and not user.date_of_birth
```

Comment:
Is there any case where user's birthday can be changed? Like, is it possible that user entered a wrong birthday, realises it later and then changes it. Or is this always coming via strong authentication.

Just thinking, because with the current code, birthday is only added but never updated.

Reply:
We should use the loa levels of high and substantial and we should be able to trust those ones. Of course if/when we are using settings with low to then we need to do some updates to code or provide some other way to update the birthday if necessary. Or contact to support :upside_down_face: 
So shortly; for now were using high `loa`s and trust'em and in future this feature might be enhanced.

Reply:
Somewhat esoteric but maybe possible scenario: foreign born person has ID assigned on 1st of January, then later gets new ID at which point actual date of birth is known and the new ID reflects that.

Reply:
Would the person be interpret as new user in profiili then?